setup.py

```diff
@@ -54,6 +54,7 @@ def package_files(directory, strip_leading):
               'Adafruit_SSD1306',
               'RPi.GPIO',
               'pyserial',
+              'progress',          
```

Comment:
You should add it to `install_requrires` and not specific to `pi`.

Reply:
Yes, and take it out of 'pc'.

Reply:
And take it out of 'dev'?    If I update the code, how do I use Github to update the pull request?

Reply:
Made the changes online

Reply:
Thanks @cfox570, this looks right. If you update your branch (i.e. push) this updates the PR automatically.

Reply:
I did the reverse. I updated through the PR and it updated my fork.  Thanks